src/auth/backchannel.ts

```diff
@@ -133,6 +138,11 @@ export type TokenResponse = {
    * The scopes associated with the token.
    */
   scope: string;
+  /**
+   * Optional authorization details when using Rich Authorization Requests (RAR).
+   * @see https://auth0.com/docs/get-started/apis/configure-rich-authorization-requests
+   */
+  authorization_details?: string;
```

Comment:
@frederikprijck: According to the [doc](https://auth0.com/docs/get-started/authentication-and-authorization-flow/client-initiated-backchannel-authentication-flow/user-authorization-with-ciba), the token response includes the authorization_details as a JSON object, not string:

<img width="826" alt="image" src="https://github.com/user-attachments/assets/33d76ef6-63db-487e-b99d-b7523e1fd50d" />